nip47/permissions/permissions.go

```diff
@@ -109,6 +109,8 @@ func scopeToRequestMethods(scope string) []string {
 		return []string{models.PAY_INVOICE_METHOD, models.PAY_KEYSEND_METHOD, models.MULTI_PAY_INVOICE_METHOD, models.MULTI_PAY_KEYSEND_METHOD}
 	case constants.GET_BALANCE_SCOPE:
 		return []string{models.GET_BALANCE_METHOD}
+	case constants.GET_BUDGET_SCOPE:
```

Comment:
Happy to instead include this in one of the other scopes if you think it makes sense.